cli/cli.ts

```diff
@@ -3881,11 +3881,14 @@ function buildCoreAsync(buildOpts: BuildCoreOptions): Promise<pxtc.CompileResult
                         if (info.pkg &&
                             info.pkg != mainPkg.config.name) delete apiInfo.byQName[infok];
                     }
+                    // Look for and read pxt snippets file
+                    const pxtsnippet = JSON.parse(mainPkg.readFile('pxtsnippets.json')) as pxt.SnippetConfig[];
```

Comment:
What happens when this file is missing?

Reply:
This returns null and will not be passed down as a file.